spec/features/item_seasons_spec.rb

```diff
@@ -3,9 +3,9 @@
 require 'rails_helper'
 
 RSpec.describe 'Item Seasons', type: :feature do
-  let(:produce_item) { ProduceItem.create(name: 'carambola', category: 'fruit') }
+  let(:produce_item) { ProduceItem.create(name: 'morango', category: 'fruit') }
```

Comment:
**suggestion:** use factories instead of creating records directly

```suggestion
  let(:produce_item) do
    FactoryBot.create(
      :produce_item, 
      name: 'morango', 
      category: 'fruit',
      item_seasons: [FactoryBot.build(:item_season)]
    )
  end
```

this would allow you to remove the before block as well

Reply:
rubocop was complaining, so I changed it a bit, it seems to work

```
RSpec.describe 'Item Seasons', type: :feature do
  let(:produce_item) do
    create(
      :produce_item,
      name: 'morango',
      category: 'fruit',
      item_seasons: [build(:item_season)]
    )
  end
```

per rubocop's docs:

<img width="317" alt="Screenshot 2022-11-09 at 11 56 14" src="https://user-images.githubusercontent.com/87862340/200863686-b3306bf2-c572-4fcd-8213-d409986dd499.png">